### ISSUE TITLE
Implement bug tracker

### ISSUE THREAD

**Issue by kontrollanten** [BeatPlus/Protonmail#114]
We should implement [Sentry](https://sentry.io), or similar, to track errors and crash reports quickly. Sentry can create GH issues based on tracked errors. Which can be a great ground to build a nice user experience.

**Comment by BeatPlus** [comment]
Great idea! I will take a look on how to implement Sentry

**Comment by kontrollanten** [comment]
Nice! I suppose we need to make that optional though. We can make it a part of the upcoming _Settings_ view.

**Comment by BeatPlus** [comment]
Definitely, it should be disabled by default _imo_

**Comment by SkyzohKey** [comment]
Please don't. Don't auto activate this by default. It would fuck the point of using Protonmail.

**Comment by kontrollanten** [comment]
No, what @BeatPlus said was that it should be deactivated by default. 

I don't think this function would "fuck the point of using ProtonMail". Not for me. I totally agree that the default mode should of course not send any data to third parts. But I think many users, at least me, don't care about an anonymous error report, without any sensitive data, is sent to a third part. Even though it should always be an active choice for the user whether to do it or not.
